src/frontend/templates/footer.html

```diff
@@ -29,11 +29,17 @@
                 <br/>
                 <small>
                     {{ if $.deploymentDetails }}
-                    <b>Cluster: </b>{{index .deploymentDetails "CLUSTERNAME" }}<br/>
-                    <b>Zone: </b>{{index .deploymentDetails "ZONE" }}<br/>
-                    <b>Pod: </b>{{index .deploymentDetails "HOSTNAME" }}
+                        {{ if index .deploymentDetails "CLUSTERNAME" }}
+                        <b>Cluster: </b>{{ index .deploymentDetails "CLUSTERNAME" }}<br/>
+                        {{ end }}
+                        {{ if index .deploymentDetails "ZONE" }}
+                        <b>Zone: </b>{{ index .deploymentDetails "ZONE" }}<br/>
+                        {{ end }}
+                        {{ if index .deploymentDetails "HOSTNAME" }}
+                        <b>Pod: </b>{{ index .deploymentDetails "HOSTNAME" }}
+                        {{ end }}
                     {{ else }}
-                    Deployment details are either still loading or failed to load.
+                    Deployment details are still loading.
```

Comment:
What if we're running on e.g. Minikube, where `deploymentDetails` is always going to be `nil`? Should we show "deployment details are still loading" if we know explicitly that it's _never_ going to appear? Why don't we just remove that section entirely, or replace it with "Deployment details unavailable"?

Reply:
Good thought.
On non-GCP clusters (e.g., Kind and minikube), the Pod name will be loaded.
![Screen Shot 2022-02-01 at 1 42 09 PM](https://user-images.githubusercontent.com/10292865/152030676-8d3ffc97-ebfb-4434-b382-00f3a3a7fc1a.png)
So let's keep this wording.

Reply:
Oh gotcha, that makes sense!!